Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN make build_ui
 ######################################
 # Prepare go_builder
 ######################################
-FROM golang:1.21-alpine as go_builder
+FROM golang:1.23.4-alpine as go_builder
```

Comment:
Does something from 1.21-alpine in go_builder end up in the final image and get scanned? I'd expect that we have to update the alpine used in the final image instead on line 21.

(unless, that ends up using the same `alpine` as go_builder, since it doesn't specify...???)

Reply:
Go dependencies are resolved and build from the alpine in line 14 as the `Run make build` is executed on line 19. 
Line 21 alpine is just for fulfilling runtime dependencies and is very light weight.

Reply:
Also, FROM alpine would anyways pull the latest stable image of alpine. So do we still need to explicitly mention the version there?